src/components/home/CardFooter.vue

```diff
@@ -5,13 +5,23 @@ const props = defineProps({
 </script>
 
 <template>
-    <div class="">
-        <slot name="logo"></slot>
+    <div class="card-footer">
+        <div class="fixed-height-logo">
+            <slot name="logo"></slot>
+        </div>
         <span class="">{{ text }}</span>
     </div>
 </template>
 
 <style scoped lang="scss">
+.card-footer {
+    display: flex;
+    gap: 5px;
 
+    justify-content: left;
 
+    .fixed-height-logo {
+        height: 30px;
+    }
+}
 </style>
```

Comment:
suggestion: c'est personnel, mais j'avoue ne pas trop aimer quand on a pas de nouvelle ligne en fin de fichier. C'est paramètrable sur vscode facilement (https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files)

Reply:
@mathieulebasic je suis d'accord. Je crois que ça se fait en eslint mais la config VScode est suffisante pour 3 devs :stuck_out_tongue: => [`573e08c` (#28)](https://github.com/leonarf/VCA4D/pull/28/commits/573e08c71fbde19e420d42e508cf84c7bb273838)